### ISSUE TITLE
Workflow problem

### ISSUE THREAD

**Issue by wensimehrp** [OpenTTD-China-Set/China-Set-Trains#30]
The typical workflow of implementing new trains is:\
```
create voxel model --> render sprites -----/---> Implement new wagon
write code --> add to .pnml index file ---/
```
The possible problem of this workflow is:
1. Different artists might use different configuration to render the model, and thus lose consistancy.
2. There are no mechanisms to make sure that when there is a change in the manifest file all models got rererndered.
3. Both the voxel models and the rendered sprites got added into the repository. This makes cloning the repo quite a pain.

I suggest renovating the current workflow by removing unnecessary files and building a rendering machenism. The workflows in [ahyangyi's](https://github.com/ahyangyi/openttd-newgrfs) and [my](https://github.com/wensimehrp/china-set-stations) repository could be used as a reference.

**Comment by jfkuayue** [comment]
One current problem of that is, the palette is not consistent.
For example, different colours for different types of CR200J use the same slot in the varying palette.
For this change, many voxels would have to be changed.

**Comment by ahyangyi** [comment]
WenSim's new approach is "one palette per subdirectory", and my approach is to use Python to control what palette file to pass to `gorender` each time.

Either way, we don't have to modify the voxel files.

**Comment by jfkuayue** [comment]
Okay, I will check then. But don't expect to be soon, I am not ready to reconstruct them.

**Comment by jfkuayue** [comment]
Is this solved now?

**Comment by wensimehrp** [comment]
Solved via #33
